packages/datadog_flutter_plugin/lib/datadog_flutter_plugin_web.dart

```diff
@@ -135,5 +129,6 @@ class _JsUser {
   });
 }
 
-@JS('setUser')
+// TODO: Need to fix this
```

Comment:
should it be a part of this PR or it is good to go with TODO item?

Reply:
Nope, I should fix it. Thanks for calling it out.